sweepbatcher/sweep_batcher.go

```diff
@@ -245,6 +252,12 @@ type Batcher struct {
 	// the caller has to update it in the source of SweepInfo (interface
 	// SweepFetcher) and re-add the sweep by calling AddSweep.
 	noBumping bool
+
+	// signMuSig2 is a custom signer. If it is set, it is used to create
+	// musig2 signatures instead of musig2SignSweep and signerClient. Note
+	// that musig2SignSweep must be nil in this case, however signerClient
+	// must still be provided, as it is used for non-coop spendings.
+	signMuSig2 SignMuSig2
```

Comment:
same here, better name it something explicit like `customMuSig2Signer`.

Reply:
Fixed